src/Captioning/Format/SubripFile.php

```diff
@@ -159,9 +159,10 @@ private function validateOptions(array $options)
     /**
      * @param string $startTimeline
      * @param string $endTimeline
+     * @param boolean $allowEqual
      * @return boolean
      */
-    private function validateTimelines($startTimeline, $endTimeline)
+    private function validateTimelines($startTimeline, $endTimeline, $allowEqual = false)
```

Comment:
Actually, this could even be true by default, as it should always be possible in my opinion.